tests/bluetooth/common/testlib/src/adv.c

```diff
@@ -60,6 +60,16 @@ int bt_testlib_adv_conn(struct bt_conn **conn, int id, uint32_t adv_options)
 	g_ctx = &ctx;
 
 	api_err = bt_le_ext_adv_create(&param, &cb, &adv);
+	if (!api_err) {
+		struct bt_data ad;
+
+		ad.type = BT_DATA_NAME_COMPLETE;
+		ad.data_len = strlen(dev_name);
+		ad.data = (const uint8_t *) dev_name;
```

Comment:
Should we add another argument so that the caller can control whether or not to set the name in ad/sd?

Reply:
As said previously this function is very specific, I don't think the user should care about that ^^